tests/fixtures/test_data/DatasetEmbedded.json

```
@@ -1,655 +1,54 @@
 {
   "items": [
     {
```

We don't really need the nesting under items, right? You could also consider defining it in yaml instead to make it a bit easier to read and write manually. But this is really just a personal preference. (Could you use an online converter if you decide to do so: https://json2yaml.com/)

```suggestion
[
    {
```